content/millennium/dstu2/individuals/patient.md

```diff
@@ -9,24 +9,30 @@ title: Patient | DSTU 2 API
 
 The following fields are returned if valued:
 
-* [Patient name](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.name){:target="_blank"}
-* [Patient id](http://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.id){:target="_blank"}
-* [Extensions including birth time, birth sex, ethnicity, and race](#extensions)
-* [Medical Record number (MRN)](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.identifier){:target="_blank"}
-* [Contact information (may include phone and email)](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.telecom){:target="_blank"}
-* [Contact person (guardian, parent or emergency)](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.contact){:target="_blank"}
-* [Gender (administrative)](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.gender){:target="_blank"}
+* [Patient ID](https://hl7.org/fhir/DSTU2/resource-definitions.html#Resource.id){:target="_blank"}
+* [name](https://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.name){:target="_blank"}
+* [Extension](#extensions)
+* [Identifier](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.identifier){:target="_blank"}
+* [Telecom](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.telecom){:target="_blank"}
+* [Contact](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.contact){:target="_blank"}
+* [Gender](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.gender){:target="_blank"}
 * [Date of Birth](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.birthDate){:target="_blank"}
 * [Deceased](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.deceased_x_){:target="_blank"}
 * [Address](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.address){:target="_blank"}
-* [Communication (preferred language)](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.communication.language){:target="_blank"}
+* [Communication language](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.communication.language){:target="_blank"}
 * [Marital status](http://hl7.org/fhir/DSTU2/patient-definitions.html#Patient.maritalStatus){:target="_blank"}
-* [Primary Care Providers](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.careProvider){:target="_blank"}
-* [Multiple Birth Information](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.multipleBirth_x_){:target="_blank"}
+* [Care Provider](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.careProvider){:target="_blank"}
+* [Multiple Birth](http://hl7.org/fhir/dstu2/patient-definitions.html#Patient.multipleBirth_x_){:target="_blank"}
```

Comment:
You'll want to double-check for https-ing here.
You got a couple above this list, but starting at `Identifier` they're http

Reply:
apparently completely missed checking this. fixing some bad links too.